common/api/summary/core-react.exports.csv

```diff
@@ -174,7 +174,7 @@ public;SettingsProvidersChangedEventArgs
 public;SettingsTabEntry
 public;SettingsTabsProvider
 internal;shallowDiffers: (a:
-internal;Size 
+public;Size 
```

Comment:
Keep internal, use inline object or https://www.itwinjs.org/reference/core-geometry/cartesiangeometry/xy/ for comparison.